elasticdl/python/keras/layers/sparse_embedding.py

```diff
@@ -0,0 +1,84 @@
+import tensorflow as tf
+from tensorflow.python.keras import backend as K
+from tensorflow.python.keras import initializers
+from tensorflow.python.ops import embedding_ops, math_ops
+
+
+class SparseEmbedding(tf.keras.layers.Layer):
```

Comment:
It seems that `tf.keras.layers.Embedding` does support SparseTensor as an input. So we provide a custom `SparseEmbedding` layer to users? As a result, users have to import SparseEmbedding to build their model.

Reply:
Yes. I have created an issue on `Tensorflow` about embedding with SparseTensor. But I don't receive any feedback. The `SparseEmbedding` is a Keras-native layer, so the user can use it with SparseTensor in any platform. In another way, we can use `tf.feature_column.embedding_column` for `SparseTensor` with variable-length data which can not use the partitioned embedding table with multiple PS instances in ElasticDL. When the custom `embedding_column` is ready in #1535, we can use it to replace `tf.feature_column.embedding_column`.